cluster/util.py

```diff
@@ -130,3 +132,38 @@ def centroid(data, method=median):
     for i in range(len(data[0])):
         out.append(method([x[i] for x in data]))
     return tuple(out)
+
+def HDcentroid(data):
+    dict_words={}
+    dict_weight={}
+    words_per_user=10 #10 words per user. This value is not used.
+    num_users_cluster=len(data)# len(data) is the number of users (user=item)
```

Comment:
Assuming this code is used with a structure other than "users" this comments does not make much sense and could be rewritten to something more generic.

Reply:
i agree